experimental/sdg-tracker/src/components/layout/components.tsx

```diff
@@ -127,3 +130,148 @@ export const SearchBar: React.FC<{
     </SearchInputContainer>
   );
 };
+
+// Country selection dropdown
+const CountrySelectContainer = styled.div`
```

Comment:
For custom DC integration, it's easier if the styles are in their own CSS. Any chance we can do that?

Reply:
I've been using the styled divs to follow the pattern set by Dan -- @dwnoble thoughts?

Reply:
for the header and footer and other reusable components i think that makes sense, but this and many other components in this project are tied to the [ant design](https://ant.design/) library, so they may not make sense to integrate with custom DC anyway

Reply:
SG. Thanks.